articles/saml/saml-sp-generic.md

```diff
@@ -154,3 +154,25 @@ Be sure to check to make sure cookies and javascript are enabled for your browse
 Check to make sure that the callback URL specified by your application in its authentication request is listed in the **Allowed Callback URLs** field in the __"Settings"__ tab of the application registered in the Auth0 Dashboard.  (In dashboard, Click on __"Apps/APIs"__ link, then on the __"Settings"__ icon to the right of the application name.)
 
 The **[http://samltool.io](http://samltool.io)** tool can decode a SAML assertion and is a useful debugging tool.
+
+# Additional information
+## Supported algorithms for signatures
+
+We currently support the following algorithms for processing XML Digital Signatures:
+
+* Canonicalization
+  * 'http://www.w3.org/2001/10/xml-exc-c14n#'
```

Comment:
The URLs should be like this: 
`[Canonical XML (XML-C14N)](http://www.w3.org/2001/10/xml-exc-c14n#)`
`[Canonical XML (XML-C14N) #WithComments](http://www.w3.org/2001/10/xml-exc-c14n#WithComments)`
and so forth